source/features/emojis-title.tsx

```diff
@@ -12,6 +12,7 @@ function init() {
 
 features.add({
 	id: 'emojis-title',
+	description: 'Don’t add a title if the emoji\'s parents already have one',
```

Comment:
Styled quote (`’`) and escaped quote (`\'`)? Choose to have only one of these.

Reply:
I read this description many times and I'm still not sure what the feature actually does.

Reply:
Sidetracking here, but it takes the `alias` attribute (provided by GH) of the emoji, and adds it as a title (if any of its closest parent doesn't provide one).

Presumably so that anyone without access to an emoji keyboard (picket) can use this info to insert emojis in comments.

Considering that GitHub already provides autocomplete for emojis in desktop view and that the `title` attribute is useless on mobile devices, I think this feature can be dropped. 🤷‍♂️

Reply:
Context:
https://github.com/sindresorhus/refined-github/issues/278
https://github.com/sindresorhus/refined-github/pull/437

I don't think I ever used it and even then you can just copy-paste the emoji.

Reply:
> I would prefer `\'`.

@bfred-it wants `’`, so we'll go with that. But should be consistent everywhere.